Lib/test/test_compile.py

```diff
@@ -1384,53 +1384,90 @@ def check_op_count(func, op, expected):
                     actual += 1
             self.assertEqual(actual, expected)
 
+        def check_num_consts(func, typ, expected):
```

Comment:
I think everywhere this is used it's called with `0` expected. Maybe just replace those calls with `check_consts(func, typ, set())` since both functions are so similar?